app/data_model/submitted_responses.py

```diff
@@ -0,0 +1,61 @@
+import boto3
+from botocore.exceptions import BotoCoreError, ConnectionError as BotoConnectionError
+from botocore.config import Config
+from app.settings import EQ_ENVIRONMENT_PREFIX, EQ_DYNAMODB_ENDPOINT
+from structlog import get_logger
+
+logger = get_logger()
+
+
+class SubmittedResponses(object):
+
+    def __init__(self):
+        config = Config(retries={'max_attempts': 1})   # Number of additional connection attempts
```

Comment:
Should `max_attempts` be ore than 1 ?

Reply:
We'll check the database is available twice as this is additional checks. It would be very unlikely it wouldn't be able to access it first time as it's AWS to AWS. Any further attempts would cause a time delay felt by the user.